src/stack_pr/cli.py

```diff
@@ -745,11 +771,15 @@ def add_cross_links(st: List[StackEntry], keep_body: bool, verbose: bool):
                 ]
             )
 
-        run_shell_command(
-            ["gh", "pr", "edit", e.pr, "-t", title, "-F", "-", "-B", e.base],
-            input="\n".join(pr_body).encode(),
-            quiet=not verbose,
-        )
+        if e.has_base():
+            run_shell_command(
+                ["gh", "pr", "edit", e.pr, "-t", title, "-F", "-", "-B", e.base or ""],
```

Comment:
Since we're checking `e.has_base()`, `or ""` shouldn't be necessary.

Reply:
mostly to shut up the type check warnings now since it can't see through e.has_base() :)